Kitodo-FileManagement/src/test/resources/kitodo_config.properties

```diff
@@ -0,0 +1,545 @@
+# * This file is part of the Kitodo Application - a Workflow tool for the support of mass digitization.
```

Comment:
Do you really want to include your local configuration file from your windows development system?